dll/shellext/zipfldr/lang/zh-TW.rc

```diff
@@ -0,0 +1,97 @@
+ *
+ * REFERENCES:  Chinese (Simplified) resource translation
+ *              Copyright 2018 Li Keqing (auroracloud4096@gmail.com)
+ */
+
+LANGUAGE LANG_CHINESE, SUBLANG_CHINESE_TRADITIONAL
+
+IDD_PROPPAGEDESTINATION DIALOGEX 0, 0, 260, 156
+STYLE DS_SHELLFONT | WS_CHILD | WS_DISABLED | WS_CAPTION
+CAPTION "選擇目標資料夾"
+FONT 9, "新細明體", 400, 0, 0x0
+BEGIN
+    LTEXT           "選擇目標資料夾",IDC_STATIC,86,12,154,8
+    EDITTEXT        IDC_DIRECTORY,86,24,157,12,ES_AUTOHSCROLL
+    PUSHBUTTON      "瀏覽...",IDC_BROWSE,189,42,54,14
+    PUSHBUTTON      "密碼",IDC_PASSWORD,189,66,54,14
+    LTEXT           "點擊 '下一步' 來解壓縮",IDC_STATUSTEXT,86,114,80,8
+    CONTROL         "",IDC_PROGRESS,"msctls_progress32",WS_BORDER,86,126,157,6
+    CONTROL         IDB_ZIPFLDR,-1,"STATIC",SS_BITMAP | WS_CHILD | WS_VISIBLE,0,0,78,138
+END
+
+IDD_PROPPAGECOMPLETE DIALOGEX 0, 0, 260, 156
+STYLE DS_SHELLFONT | WS_CHILD | WS_DISABLED | WS_CAPTION
+CAPTION "解壓縮完成"
+FONT 9, "新細明體", 400, 0, 0x0
+BEGIN
+    LTEXT           "檔案已被解壓縮至下列目錄：",IDC_STATIC,86,12,157,18
+    LTEXT           "%s",IDC_DESTDIR,86,36,157,8
+    CONTROL         "顯示被解壓縮檔案",IDC_SHOW_EXTRACTED,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,86,66,81,10
+    LTEXT           "點擊完成以繼續。",IDC_STATIC,86,84,84,8
+    CONTROL         IDB_ZIPFLDR,-1,"STATIC",SS_BITMAP | WS_CHILD | WS_VISIBLE,0,0,78,138
+END
+
+IDD_CONFIRM_FILE_REPLACE DIALOGEX 0, 0, 273, 56
+STYLE DS_MODALFRAME | DS_SHELLFONT | WS_POPUP | WS_CAPTION | WS_SYSMENU
+CAPTION "確認替換檔案"
+FONT 9, "新細明體", 400, 0, 0x1
+BEGIN
+    DEFPUSHBUTTON   "是(&Y)",IDYES,6,36,62,14
+    PUSHBUTTON      "全部為是(&T)",IDYESALL,72,36,62,14
+    PUSHBUTTON      "否(&N)",IDNO,138,36,62,14
+    PUSHBUTTON      "取消",IDCANCEL,204,36,62,14
+    ICON            "",IDC_EXCLAMATION_ICON,6,6,24,22
+    LTEXT           "",IDC_MESSAGE,36,6,228,24
+END
+
+IDD_PASSWORD DIALOGEX 0, 0, 243, 62
+STYLE DS_SETFONT | DS_FIXEDSYS | WS_POPUP | WS_CAPTION | WS_SYSMENU
+CAPTION "需要密碼"
+FONT 9, "新細明體", 400, 0, 0x1
+BEGIN
+    EDITTEXT        IDC_PASSWORD,72,42,108,13,ES_PASSWORD | ES_AUTOHSCROLL
+    DEFPUSHBUTTON   "確定",IDOK,186,6,50,14
+    PUSHBUTTON      "略過檔案",IDSKIP,186,24,50,14
+    PUSHBUTTON      "取消",IDCANCEL,186,42,50,14
+    LTEXT           "Static",IDC_MESSAGE,24,6,156,30
+    LTEXT           "密碼︰",IDC_STATIC,24,44,44,8
+END
+
+STRINGTABLE
+BEGIN
+    IDS_COL_NAME "名稱"
+    IDS_COL_TYPE "類型"
+    IDS_COL_COMPRSIZE "壓縮大小"
+    IDS_COL_PASSWORD "密碼"
+    IDS_COL_SIZE "大小"
+    IDS_COL_RATIO "壓縮率"
+    IDS_COL_DATE_MOD "修改日期"
+    IDS_YES "是"
+    IDS_NO "否"
+    IDS_ERRORTITLE "壓縮的 (zipped) 資料夾發生錯誤"
+    IDS_CANTSTARTTHREAD "Cannot start thread to compress files."
+    IDS_NOFILES "The specified directory '%s' is empty, so Compressed (zipped) Folders cannot add it to the archive."
+    IDS_CANTCREATEZIP "Failed to create a compressed folder '%s' (Error Code: %d)."
+    IDS_CANTREADFILE "無法讀取檔案 '%s'."
+    IDS_EXTRACTING "正在解壓縮..."
+
+    IDS_WIZ_TITLE "解壓縮精靈"
```

Comment:
may I ask why not translate as 「解壓縮向導」？

Reply:
1. The term 'Wizard' is translated to 「精靈」 instead of 「向導」(嚮導) in Traditional Chinese. The term and its translation can be found on Mic**t's terminology and Traditional Chinese Wikipedia.
2. Consistency (Compare the uses of the term 「[精靈](https://git.reactos.org/?p=reactos.git&a=search&h=1d1bb5332135c192166783735cd35447ae699f0d&st=grep&s=%E7%B2%BE%E9%9D%88)」 and 「[嚮導](https://git.reactos.org/?p=reactos.git&a=search&h=1d1bb5332135c192166783735cd35447ae699f0d&st=grep&s=%E5%9A%AE%E5%B0%8E)」 in grep)

Reply:
Oh, sorry I don't know this before 🤣. Then it's fine ofcourse !
Thank you for explain this.

Reply:
> 
> 
> (Sorry for the late reply because I received the comment in midnight)
> 
>     1. The term 'Wizard' is translated to 「精靈」 instead of 「向導」(嚮導) in Traditional Chinese. The term and its translation can be found on Mic**t's terminology and Traditional Chinese Wikipedia.
> 
>     2. Consistency (Compare the uses of the term 「[精靈](https://git.reactos.org/?p=reactos.git&a=search&h=1d1bb5332135c192166783735cd35447ae699f0d&st=grep&s=%E7%B2%BE%E9%9D%88)」 and 「[嚮導](https://git.reactos.org/?p=reactos.git&a=search&h=1d1bb5332135c192166783735cd35447ae699f0d&st=grep&s=%E5%9A%AE%E5%B0%8E)」 in grep)

Why censor the word Microsoft?

Reply:
> > (Sorry for the late reply because I received the comment in midnight)
> > ```
> > 1. The term 'Wizard' is translated to 「精靈」 instead of 「向導」(嚮導) in Traditional Chinese. The term and its translation can be found on Mic**t's terminology and Traditional Chinese Wikipedia.
> > 
> > 2. Consistency (Compare the uses of the term 「[精靈](https://git.reactos.org/?p=reactos.git&a=search&h=1d1bb5332135c192166783735cd35447ae699f0d&st=grep&s=%E7%B2%BE%E9%9D%88)」 and 「[嚮導](https://git.reactos.org/?p=reactos.git&a=search&h=1d1bb5332135c192166783735cd35447ae699f0d&st=grep&s=%E5%9A%AE%E5%B0%8E)」 in grep)
> > ```
> 
> Why censor the word Microsoft?

just dunno whether saying Microsoft is allowed lol